changes/1ddf4c80ca6c07897e48bcad31c67d57.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Fix bug where trigger name and doc updates set via runtnodes did not go through
```

Comment:
`Fix a bug where trigger name and doc updates set via syn:trigger nodes did not persist.`

Reply:
``syn:trigger``